receiver/kafkareceiver/json_unmarshaler.go

```diff
@@ -0,0 +1,43 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package kafkareceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver"
+import (
+	"encoding/json"
+	"time"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+)
+
+type jsonLogsUnmarshaler struct {
+}
+
+func newJSONLogsUnmarshaler() LogsUnmarshaler {
+	return &jsonLogsUnmarshaler{}
+}
+
+func (r *jsonLogsUnmarshaler) Unmarshal(buf []byte) (plog.Logs, error) {
+	// create a new Logs struct to be populated with log data and returned
+	p := plog.NewLogs()
+
+	// get json logs from the buffer
+	jsonVal := map[string]interface{}{}
+	if err := json.Unmarshal(buf, &jsonVal); err != nil {
```

Comment:
Could I ask you to use `jsoniter` instead?

It has improved performance over the standard lib json encoding.